paddle/fluid/operators/fused_embedding_fc_lstm_op.cc

```diff
@@ -93,11 +93,8 @@ void FusedEmbeddingFCLSTMOp::InferShape(
   ctx->SetOutputDim("Cell", out_dims);
   ctx->ShareLoD("Ids", "Hidden");
   ctx->ShareLoD("Ids", "Cell");
-  int xx_width;
-  if (ctx->Attrs().Get<bool>("use_seq")) {
-    xx_width = wh_dims[1];
-  } else {
-    xx_width = x_dims[1] > wh_dims[1] ? wh_dims[1] : x_dims[1];
+  int xx_width = wh_dims[1];
```

Comment:
Does it mean "x_dims[1] > wh_dims[1]" is always true？

Reply:
Why keep xx_width always be wh_dims[1]?

Reply:
@tensor-tang Good catch! It should be fixed now. Thanks!

@hshen14 This op is based on fusion_lstm_op.cc . And in this fusion lstm op there were two paths of  doing  Embedding*W_x operation depending on the dims of buffers. I think that was matter of optimization (I didn't really understand the reasons) . Anyway In this op: fused_embedding_fc_lstm We do not perform embedding_W_x , but just look it up in an array , so there is no need to evaluate condition that was removed. It is even worse, op was crashing when BatchCompute path was executed with this condition enabled.